pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/CohortTagMigrator.java

```diff
@@ -25,46 +20,11 @@ public CohortTagMigrator(String index, String realm, CohortTagDao cohortTagDao)
     @Override
     protected Map<String, Object> getDataByRealm() {
         Map<String, List<CohortTag>> cohortTags = cohortTagDao.getCohortTagsByInstanceName(realm);
-        log.info("Retrieved {} cohort tags from realm {}", cohortTags.size(), realm);
-        updateCohortTagsIfRequired(cohortTags);
-        addOldOsteoTagsToNewOsteo(cohortTags, realm);
-        return (Map) cohortTags;
-    }
-
-    private void addOldOsteoTagsToNewOsteo(Map<String, List<CohortTag>> cohortTags, String realm) {
-        if (realm.equalsIgnoreCase(NEW_OSTEO_INSTANCE_NAME)) {
-            OldOsteoCohortTagsRetriever oldOsteoRetriever = new OldOsteoCohortTagsRetriever(new CohortTagDaoImpl());
-            Map<String, List<CohortTag>> oldOsteoTags = oldOsteoRetriever.retrieve();
-            log.info("Retrieved {} cohort tags via OldOsteoCohortTagsRetriever", oldOsteoTags.size());
-            oldOsteoTags.forEach((guid, records) -> {
-                if (cohortTags.containsKey(guid)) {
-                    List<CohortTag> mergedCohorts = Stream.concat(cohortTags.get(guid).stream(), records.stream())
-                            .collect(Collectors.toList());
-                    cohortTags.put(guid, mergedCohorts);
-                } else {
-                    // since these two realms share the same ES index, also add old osteo tags for ptps
-                    // that are only in old osteo realm
-                    cohortTags.put(guid, records);
-                }
-            });
-        }
-    }
-
-    private void updateCohortTagsIfRequired(Map<String, List<CohortTag>> cohortTags) {
+        int tagsFromRealm = cohortTags.size();
         AdditionalCohortTagsRetriever.fromRealm(realm)
```

Comment:
Let's see if I understand this: whenever we're dealing with os1 or os2, we'll get back both tags?

Reply:
Yes. This is the old code refactored. No intended behavior change.